index.js

```diff
@@ -32,22 +30,27 @@ class KeyState extends q.DesktopApp {
   // eslint-disable-next-line class-methods-use-this
   async run() {
     KeyState.log('Running...');
-    const key = this.config.key || this.keys.CapsLock.code;
-    return KeyState.getKeyStatus(key)
-      .then((status) => {
-        const enabled = status === 'True';
-        return new q.Signal({
-          points: [
-            [this.generatePoint(enabled)],
-          ],
-          name: `${key} State`,
-          message: `${key} is ${enabled ? 'enabled' : 'disabled'}`,
-        });
-      }).catch((err) => {
-        // TODO: Introduce an error signal
-        // https://www.daskeyboard.io/applet-development/signals#signal-options
-        KeyState.log(err, 'error');
-      });
+
+    this.pollingInterval = this.config.pollingInterval || 1000;
+    this.colors = {
+      on: this.config.colorOn || '#FF0000',
+      off: this.config.colorOff || '#00FF00',
+    };
+    var key = this.config.key || this.keys.CapsLock.code;
```

Comment:
If this can't be `const`, let's use `let` unless we specifically want the scoping behavior of `var` here but I don't think we need that.

Reply:
Actually, was this changed because the `const` value may not be initialized and thus the runtime threw an error?

Reply:
I honestly have not written enough Javascript to know the difference between the three. I am literally learning for this.

Do you want me to add the edits, or will you merge to a branch, add the edits, and merge?